src/django_otp_webauthn/models.py

```diff
@@ -85,6 +78,21 @@ class Format(models.TextChoices):
     important because it needs to be byte-for-byte identical in order to verify
     the signature."""
 
+    @classmethod
+    def check(cls, **kwargs):
+        errors = super().check(**kwargs)
+
+        if not hasattr(cls, 'credential'):
+            errors.append(
+                checks.Error(
+                        "missing 'credential' field",
+                        hint=f"Add a field named 'credential' as a OneToOneField relating to {get_credential_model_string()}.",
+                        obj=cls,
+                        id="otp_webauthn.E001",
```

Comment:
**Suggestion:** can we make this refer to a constant in `checks.py`? I like to keep an overview of what error codes we have defined in one place.

Reply:
Yep, can't believe I didn't think to do that in the first place.

Reply:
@Stormheg Somehow I had totally missed that you already had a `checks.py` with constants in it for current errors. Is there a scheme you are using for choosing the error codes?

Reply:
I haven't thought about it much. Looking at it now, it looks like I grouped similar checks together

> ERR_NO_RP_ID = "otp_webauthn.E010"
> ERR_NO_RP_NAME = "otp_webauthn.E011"
>ERR_UNSUPPORTED_COSE_ALGORITHM = "otp_webauthn.E020"
>ERR_NO_COSE_ALGORITHMS_CONFIGURED = "otp_webauthn.E021"
>...

I'd just add to the list of constants. The next available code following this scheme would be `otp_webauthn.E050`

Reply:
Sounds good, I'll have this ready to go at some point this weekend. Not much work to do, just gotta find time around family, etc.